cliboa/scenario/transform/file.py

```diff
@@ -773,3 +774,46 @@ def execute(self, *args):
                 raise InvalidParameter(
                     "'format' must set one of the followings [tar, zip]"
                 )
+
```

Comment:
I don't think this method belongs to FileBaseTransform.
Why don't you create a new module named system.py ( maybe ?),
and move the mehod.

If so , FileBaseTransform does not have to be a parent class of this.
Change parent class to BaseStep instead

Reply:
Moved to transform/system thank you for the review